platform-emf/src/main/java/com/softicar/platform/emf/action/EmfPromptActionInput.java

```diff
@@ -92,7 +91,7 @@ public void appendRow(IDisplayString title, IDomNode node) {
 	@Override
 	public DomTextArea appendTextArea(IDisplayString title) {
 
-		EmfMultilineStringInput textArea = new EmfMultilineStringInput();
+		DomTextArea textArea = new DomTextArea().setRowCount(5);
```

Comment:
I think we should extract that `5` to a descriptively-named constant.

Reply:
- I see no benefit in making this a constant, this number has no "semantics" and is not reused.
- Actually, we should probably remove all those calls and make this over CSS.